pkg/collector/corechecks/cluster/ksm/kubernetes_state_transformers.go

```diff
@@ -298,11 +298,21 @@ func cronJobLastScheduleTransformer(s aggregator.Sender, name string, metric ksm
 
 // jobCompleteTransformer sends a service check based on kube_job_complete
 func jobCompleteTransformer(s aggregator.Sender, name string, metric ksmstore.DDMetric, hostname string, tags []string) {
+	for i, tag := range tags {
+		if tag == "condition:true" {
+			jobMetric(s, metric, ksmMetricPrefix+"job.completion.succeeded", hostname, append(tags[:i], tags[i+1:]...))
```

Comment:
Should there be a `break` here?

Reply:
Also, should we update the description of the function to reflect the new reporting?

Reply:
Definitively yes.
I’ve added the `break` and updated the function comment.